<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C</class>
 <widget class="QMainWindow" name="C">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop Crop PDF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ccp_tab">
     <attribute name="title">
      <string>CCP</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="directory_path_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Manga Directory</string>
       </property>
      </widget>
      <widget class="QPushButton" name="manga_browse_button_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="crop_button_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CROP</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="separate_folder_check_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Serparate Cropped Folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf_button_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MAKE PDF</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="vol_extension_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Extension Name (vol/delux/vigbig)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="manga_name_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Manga Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="starting_vol_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>130</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Starting From</string>
       </property>
      </widget>
      <widget class="QPushButton" name="select_vol_button_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Selct Volumes</string>
       </property>
      </widget>
      <widget class="QLabel" name="set_vol_label_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>How Many Volumes In a Set</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="volume_directory_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Volume Directory</string>
       </property>
      </widget>
      <widget class="QPushButton" name="volume_browse_button_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="set_vol_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QPushButton" name="ltr_check_button_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LTR</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rtl_check_button_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RTL</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGraphicsView" name="bg2">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-20</y>
         <width>521</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <zorder>bg2</zorder>
      <zorder>directory_path_2</zorder>
      <zorder>manga_browse_button_2</zorder>
      <zorder>crop_button_2</zorder>
      <zorder>separate_folder_check_2</zorder>
      <zorder>pdf_button_2</zorder>
      <zorder>vol_extension_2</zorder>
      <zorder>manga_name_2</zorder>
      <zorder>starting_vol_2</zorder>
      <zorder>select_vol_button_2</zorder>
      <zorder>set_vol_label_2</zorder>
      <zorder>volume_directory_2</zorder>
      <zorder>volume_browse_button_2</zorder>
      <zorder>set_vol_2</zorder>
      <zorder>ltr_check_button_2</zorder>
      <zorder>rtl_check_button_2</zorder>
     </widget>
     <widget class="QLabel" name="AppName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Showcard Gothic</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CROP CROP PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="copyright_label">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>280</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Copyright: Saad Ibne Sohael; saad.raj.bd@gmail.com; https:www.facebook.com/isaadsohael</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="bg1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <zorder>bg1</zorder>
     <zorder>frame</zorder>
     <zorder>AppName</zorder>
     <zorder>copyright_label</zorder>
    </widget>
    <widget class="QWidget" name="blank_page_tab">
     <attribute name="title">
      <string>Add Blank Page</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>243</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Showcard Gothic</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CROP CROP PDF</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>481</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Width</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <property name="currentText">
            <string>White Page</string>
           </property>
           <item>
            <property name="text">
             <string>White Page</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Black Page</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Save as</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_4">
           <property name="currentText">
            <string>JPG</string>
           </property>
           <item>
            <property name="text">
             <string>JPG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>JPEG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PNG</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Page</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="bg3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <zorder>bg3</zorder>
     <zorder>label_11</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuChange_Theme">
     <property name="title">
      <string>Change Theme</string>
     </property>
     <addaction name="actionWhite_Theme"/>
     <addaction name="actionBlack_Theme"/>
    </widget>
    <addaction name="menuChange_Theme"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionBlankPage">
   <property name="text">
    <string>Add Blank Page</string>
   </property>
  </action>
  <action name="actionManual_Crop">
   <property name="text">
    <string>Manual Crop</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionWhite_Theme">
   <property name="text">
    <string>White Theme (Current)</string>
   </property>
  </action>
  <action name="actionBlack_Theme">
   <property name="text">
    <string>Black Theme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
